chapters/dashboard/index.md

```diff
@@ -1,186 +1,66 @@
-# Using Dashboard to Monitor Logs and View Flow
+# Jina Dashboard
 
-**Dashboard** (beta) offers the insight of running tasks in Jina. With dashboard, one can analyze logs, design flows and view Jina Hub images.
+**Jina Dashboard** is a visual interface to facilitate Development with Jina.
 
-![flow demo](flow-demo.gif)
+In order do use the Dashboard visit https://dashboard.jina.ai
```

Comment:
Maybe wrap the link so it's not visible? [Something like this](https://dashboard.jina.ai/)?

Reply:
Good idea!